### ISSUE TITLE
Add `edit` / `signal issue` links

### ISSUE THREAD

**Issue by alrra** [sonarwhal/sonarwhal.com#128]
For documentation pages:

 * `edit`- should link to the corresponding `.md` file from the sonar repository. 

    E.g.: https://remysharp.com/2017/05/01/getting-the-latest-emoji-on-my-mac

   ![](https://user-images.githubusercontent.com/1223565/26877397-7a9bbedc-4b92-11e7-9635-294d5e475209.gif)

* `signal issue` - should open new issue in the main sonar repository with the title of the documentation page, e.g.: https://github.com/MicrosoftEdge/Sonar/issues/new?title=[docs]%20Issue%20with%20%27%3Cpage%20title%3E%27 (we could even add the selected text as the body of the issue)

**Comment by molant** [comment]
@ststimac for you :)

**Comment by ststimac** [comment]
Does the second part of this (opening a new issue in sonar repo w/the documentation title) require some JS trickery?

**Comment by molant** [comment]
It will be on the backend and @qzhou1607 can probably help with that once the UI is done :)

**Comment by qzhou1607** [comment]
If only opening a new issue page is involved, all it needs is replacing `title` with the right information, which can be done in the templates rendered by hexo. But yeah I can definitely help :).
